ipynb/releases/ReleaseNotes_v16.12.ipynb

```diff
@@ -0,0 +1,778 @@
+      "2016-12-15 17:15:15,672 INFO    : TestEnv      :    CATAPULT_HOME: /home/patbel01/Code/lisa/tools/catapult\n",
+      "2016-12-15 17:15:15,672 INFO    : TestEnv      : Loading board:\n",
+      "2016-12-15 17:15:15,672 INFO    : TestEnv      :    /home/patbel01/Code/lisa/libs/utils/platforms/pixel.json\n",
+      "2016-12-15 17:15:15,673 INFO    : TestEnv      : Devlib modules to load: [u'bl', u'cpufreq']\n",
+      "2016-12-15 17:15:15,674 INFO    : TestEnv      : Connecting Android target [DEFAULT]\n",
+      "2016-12-15 17:15:15,674 INFO    : TestEnv      : Connection settings:\n",
+      "2016-12-15 17:15:15,674 INFO    : TestEnv      :    None\n",
+      "2016-12-15 17:15:15,733 INFO    : android      : ls command is set to ls\n",
+      "2016-12-15 17:15:16,068 INFO    : TestEnv      : Initializing target workdir:\n",
+      "2016-12-15 17:15:16,068 INFO    : TestEnv      :    /data/local/tmp/devlib-target\n",
+      "2016-12-15 17:15:16,555 INFO    : TestEnv      : Topology:\n",
+      "2016-12-15 17:15:16,556 INFO    : TestEnv      :    [[0, 1], [2, 3]]\n",
+      "2016-12-15 17:15:16,635 INFO    : TestEnv      : Loading default EM:\n",
+      "2016-12-15 17:15:16,636 INFO    : TestEnv      :    /home/patbel01/Code/lisa/libs/utils/platforms/pixel.json\n",
+      "2016-12-15 17:15:16,637 INFO    : TestEnv      : Set results folder to:\n",
+      "2016-12-15 17:15:16,637 INFO    : TestEnv      :    /home/patbel01/Code/lisa/results/20161215_171516\n",
+      "2016-12-15 17:15:16,638 INFO    : TestEnv      : Experiment results available also in:\n",
+      "2016-12-15 17:15:16,638 INFO    : TestEnv      :    /home/patbel01/Code/lisa/results_latest\n"
+     ]
+    }
+   ],
+   "source": [
+    "from env import TestEnv\n",
+    "\n",
+    "te = TestEnv({\n",
+    "        'platform'     : 'android',\n",
+    "        'board'        : 'pixel',\n",
+    "        'ANDROID_HOME' : '/home/patbel01/Code/lisa/tools/android-sdk-linux/'\n",
+    "    }, force_new=True)\n",
+    "target = te.target"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true
+   },
+   "source": [
+    "## Added UiBench workload"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A new Android benchmark has been added to run UiBench provided tests.\n",
+    "Here is a notebook which provides an example of how to run this test on your\n",
+    "android target:\n",
+    "\n",
+    "   https://github.com/ARM-software/lisa/blob/master/ipynb/examples/android/benchmarks/Android_UiBench.ipynb"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true
+   },
+   "source": [
+    "# Tests"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true
+   },
+   "source": [
+    "## intial version of the preliminary tests"
```

Comment:
It's also actually mine... but my recent abuse of drugs should have obfuscated my mind :-D